WorkloadManagementSystem/Service/MatcherHandler.py

```diff
@@ -118,7 +119,9 @@ def export_getActiveTaskQueues():
 
 ##############################################################################
   types_getMatchingTaskQueues = [dict]
+  # int keys are cast into str
 
+  @ignoreEncodeWarning
```

Comment:
Apply also to `export_getActiveTaskQueues` (also for client, I think)

Reply:
OK